thirdeye-server/src/main/java/ai/startree/thirdeye/resources/AlertResource.java

```diff
@@ -257,4 +267,26 @@ private long safeEndTime(final @Nullable Long endTime) {
     }
     return endTime;
   }
+
+  @GET
+  @Timed
+  @Path("{id}/stats")
+  @Produces(MediaType.APPLICATION_JSON)
+  public Response getAnalytics(
+      @ApiParam(hidden = true) @Auth ThirdEyePrincipal principal,
+      @PathParam("id") final Long id,
+      @Context UriInfo uriInfo
+  ) {
+    ensureExists(id);
+    final MultivaluedMap<String, String> queryParameters = uriInfo.getQueryParameters();
+    final Predicate alertPredicate = Predicate.EQ("detectionConfigId", id);
+    final DaoFilter filter;
+    if (queryParameters.isEmpty()) {
+      filter = new DaoFilter().setPredicate(alertPredicate);
+    } else {
+      filter = new DaoFilterBuilder(AnomalyResource.API_TO_INDEX_FILTER_MAP).buildFilter(queryParameters);
```

Comment:
This logic is for general query param processing in CrudResource. Can you follow a simpler example: https://github.com/startreedata/thirdeye/blob/master/thirdeye-server/src/main/java/ai/startree/thirdeye/resources/DataSourceResource.java#L187

Reply:
Restricted the allowed query params for filtering to only enumerationItem.id, startTime, endTime.